tests/README.md

```diff
@@ -54,62 +56,25 @@ To write an `istio.io` test, follow these steps:
    Fix the errors, if any, by updating the corresponding command in the `index.md` file and
    then regenerate the snips.
 
-1. Pick an appropriate location under the `tests/` directory for your new test.
-
-1. Create Go boilderplate that will invoke your test bash script using the following pattern:
-
-    ```golang
-    package <your-test-package>
-
-    import (
-        "testing"
-
-        "istio.io/istio/pkg/test/framework"
-
-        "istio.io/istio.io/pkg/test/istioio"
-    )
-
-    func Test<your-test>(t *testing.T) {
-        framework.
-            NewTest(t).
-            Run(istioio.NewBuilder("<your-test-name>").
-                Add(istioio.Script{
-                    Input: istioio.Path("scripts/<your-bash-script>.sh"),
-                }).
-                Defer(istioio.Script{
-                    Input: istioio.Inline{
-                        FileName: "cleanup.sh",
-                        Value: `
-    set +e # ignore cleanup errors
-    source ${REPO_ROOT}/content/en/docs/<your-snips-dir>/snips.sh
-    <your cleanup steps>`,
-                    },
-                }).
-                Build())
-    }
-    ```
-
-    NOTE: This Go boilerplate is a temporary requirement. It will not be needed in the future.
-    See https://docs.google.com/document/d/1r_NoxatNjzPsw0eXr6_9W0rqlkfelt7QfN2y5TF0yTo/edit#.
-
-1. Create your test bash script in the `scripts/` subdirectory.
+1. Create your test bash script `test.sh` next to the `snips.sh` you have just generated. Scripts that have other names will be ignored.
```

Comment:
@frankbu Is `make snips` automatically called from `make gen`? If not it should be. The `gen` target is used by CI to determine if all generated files are up-to-date.

Reply:
Yes it is.